src/app/components/auth/login/login.component.ts

```diff
@@ -1,34 +1,60 @@
-import { Component } from '@angular/core';
-import { FormBuilder, FormGroup, Validators } from '@angular/forms';
-import { Router } from '@angular/router';
-import { RouterModule } from '@angular/router';
-import { ReactiveFormsModule } from '@angular/forms';
-import { InputTextModule } from 'primeng/inputtext';
-import { CardModule } from 'primeng/card';
-import { ButtonModule } from 'primeng/button';
+import { Component } from "@angular/core";
+import { FormBuilder, FormGroup, Validators } from "@angular/forms";
+import { Router } from "@angular/router";
+import { RouterModule } from "@angular/router";
+import { ReactiveFormsModule } from "@angular/forms";
+import { InputTextModule } from "primeng/inputtext";
+import { CardModule } from "primeng/card";
+import { ButtonModule } from "primeng/button";
+import { HttpClient } from "@angular/common/http";
+import { AuthService } from "src/app/services/auth.service";
```

Comment:
there is a convention that the 3rd party import to be the first and imports from project's context to be the second separated by an empty lines
```suggestion
import { HttpClient } from "@angular/common/http";

import { AuthService } from "src/app/services/auth.service";
```